modules/specifying-the-environment-variables-in-the-securedcluster-cr.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * cloud_service/installing_cloud_ocp/configuring-the-proxy-for-collector-in-rhacs-cloud-service.adoc
+:_content-type: PROCEDURE
+[id="specifying-the-environment-variables-in-the-securedcluster-cr_{context}"]
+= Specifying the environment variables in the SecuredCluster CR
+
+[role="_abstract"]
+To configure an egress proxy, you can either use the cluster-wide {osp} proxy or specify the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables within the SecuredCluster Custom Resource (SecuredCluster CR) configuration file to ensure proper use of the proxy and bypass for internal requests within the specified domain.
+
+The proxy configuration is applied to all running services, Sensor, Collector, Admission Controller and Scanner.
+
+
+.Procedure
+
+* Specify the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables under the customize specification in the SecuredCluster CR configuration file:
++
+.Specifying environment variables
+====
```

Comment:
I don't think the code block needs to also be in a admonition block. You can just use the 
```
----
code 
----
```
code block format